### PR TITLE
Feat: Added Cards

### PR DESCRIPTION
**Pull request by anjali1102**
>Cards
-  Horizontal Card
- Vertical Card
- Text Overlay
- Text Dismal Card
 
 
 [PREVIEW HERE 👈👈](https://rapidui-git-cards-anjali1102.vercel.app/components/Card/card.html)

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]
Code looks clean . Good to go

**Review by manismk — APPROVED** [review]
Looks good

File: components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  position: relative;
+}
+.card-vertical img {
+  height: 15rem;
+  width: 15rem;
+  border-radius: 10px 10px 0 0;
+}
+
+.card-badge {
+  background-color: #c45500;
```
Comment (jeetsdev) [inline components/Card/card.css]: Use variables here.

File: components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  color: white;
```
Comment (jeetsdev) [inline components/Card/card.css]: Use variables here.

File: components/Card/card.css [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+.card-image {
+  border-radius: 10px 10px 0 0;
+  max-width: 100%;
+  object-fit: cover;
+}
+
+.card-image {
+  background-color: rgb(202, 199, 199);
```
Comment (jeetsdev) [inline components/Card/card.css]: Use variables here.

File: components/Card/card.css [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+  padding-right: 0.45rem;
+}
+.actual-price {
+  text-decoration: line-through;
+  color: var(--ruler);
+  padding-right: 0.45rem;
+}
+.price-percentage {
+  color: var(--green);
+}
```
Comment (jeetsdev) [inline components/Card/card.css]: Maintain spacing consistency.

File: components/Card/card.css [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+.fa-heart-o:hover {
+  background-color: rgb(190, 190, 190);
```
Comment (jeetsdev) [inline components/Card/card.css]: Use colors varibles.

**Review by jeetsdev — CHANGES_REQUESTED** [review]

File: components/Card/card.css (hunk 1 above)
Comment (anjali1102) [reply components/Card/card.css]: okkey

**Review by jeetsdev — APPROVED** [review]

**Pull request merged**